tox.ini

```diff
@@ -8,7 +8,7 @@ max-complexity = 10
 [testenv]
 usedevelop=True
 deps=
-    pytest
+    pytest<3.3.0 # pinned due to bad pypi uploads
```

Comment:
Comment broke test
```
line 96, in __init__
    requirement_string[e.loc:e.loc + 8]))
InvalidRequirement: Invalid requirement, parse error at "'# pinned'"
```

Reply:
thanks for the note, but as far as i can tell pypi metadata was fixed so this pr is no longer needed to begin with